### DISCUSSION TITLE
global_town_settings.is_nation_ally_spawning_requiring_public_status

### DISCUSSION THREAD

**Discussion by ProblemsSender** [TownyAdvanced/Towny#5410] [Q&A] [answered]
This line... I don't get it, is it making the public status setting to be not up to the town owner's choice when set to false?

So when a resident of the town wants to do /t spawn, they don't have to be required the town's public status to be true? As this applied to another town that is within the same nation and the town that's not a member of the same nation must be treated as Ally to do /t spawn freely?

Then, the town's public status is applied to outsiders only, do I get it right? Once this `global_town_settings.is_nation_ally_spawning_requiring_public_status` statement is set to be true, it will also apply to the resident from the town level to the national and ally level?

Thank you.

**Comment by LlmDl** [comment, marked as answer]
```
  # When set to true both nation and ally spawn travel will also require the target town to have their status set to public.
  is_nation_ally_spawning_requiring_public_status: 'false'
  ```
Town members can spawn to their own town, whether public is on or off. Nation members and allies also can spawn to a town, whether public is on or off, unless this option is set to true. This changes it from being ally-friendly, to something a mayor can choose via `/t toggle public`.
